aws-sagemaker-hosted-scorer-cpp/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.8
+
+RUN apt-get -y update && pip3 install datatable \
```

Comment:
@Susankha better to have 2layers:

- first one is about system update (apt-get update, ...) - see Docker best practices - how to also cleanup `apt-get` cache to avoid spurious apt-get cache : https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run
- 2nd layer contains all pip install - i would even recommend to keep requirements file and do `pip install -r requirements.txt`

Reply:
Done